README.md

```diff
@@ -1,42 +1,80 @@
 # Immersive Web Community Group Proposals Repo
 
 ## What is this repo for?
-This repo serves as a discussion board for early proposals for additions and improvements to Immersive Web functionality.
+
+This repo serves as a discussion board for additions and improvements to Immersive Web standards.
+
+While much of our work to date has been on the WebXR Device API, this repo also holds proposals for features that could end up in other standards.
+
+If your proposal isn't in some way related to Immersive Web technologies like Virtual/Augmented/Mixed Reality then head over to the [W3C Community and Business Groups](https://www.w3.org/community/) to find an appropriate group. The [Web Incubator Community Group (WICG)](https://wicg.io/) is also a good general-purpose venue.
+
+## Code of Conduct
+
+**TL;DR: Treat other participants with respect or get banned.**
+
+This repository is for *open discussion of ideas* for making the Immersive Web better and we foster by an environment where all discussion is respectful. Every person participating in the community group is expected to follow the [Code of Conduct](https://github.com/immersive-web/community-resources/blob/master/CODE_OF_CONDUCT.md).
+
+## How does a feature idea become a standard?
+
+An immersive web feature goes through several steps to move from idea to standard:
+
+- File a Proposals Issue
+- Find Consensus for the Idea
+- Recruit Feature Advocates
```

Comment:
It's hard to tell if "feature advocate" is an explicit title/role, and if it's the same as the "feature champion" you mention below.  Do you just mean "members interested in this issue"?